src/main/java/jdbchomework/console/AbstractMenu.java

```diff
@@ -26,7 +31,19 @@ public static void outputSplitLine() {
         }
     }
 
-    public String getValidInputFromUser(String message, InputType type) {
+    protected int readInt(String message) {
+        return Integer.parseInt(getValidInputFromUser(message, InputType.INTEGER));
+    }
+
+    protected String readString(String message) {
+        return getValidInputFromUser(message, InputType.STRING);
+    }
+
+    protected void printSplitLine() {
+        System.out.println("**************************************************************");
+    }
+
+    protected String getValidInputFromUser(String message, InputType type) {
         System.out.println(message);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)